grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -728,7 +729,46 @@ class QueryService implements Status {
         Collection result = Collections.emptyList()
 
         if (ids.size() > 0) {
-            result = Issue.findAllByIdInList(ids)
+            result = findIssuesSearchItems(ids)
+        }
+        result
+    }
+
+    /**
+     * Find issues and create DTOs to narrow the amount of data used in the search process
+     * @param issueIds
+     * @return
+     */
+    List<Issue> findIssuesSearchItems(ArrayList<Integer> issueIds) {
+        final String query = "SELECT i.id id, " +
+                "i.project_key projectKey, " +
+                "i.type type, " +
+                "i.status status, " +
+                "i.summary summary, " +
+                "i.reporter reporter, " +
+                "i.update_date updated, " +
+                "i.expiration_date expirationDate " +
+                "FROM issue i " +
+                "WHERE i.id IN (" + issueIds.join(",") + ")"
```

Comment:
Is there a limit to the number of ids here? I know that oracle has a 1000 item limit, not sure if mysql or cloudsql has the same kind of thing.

Reply:
MySQL has 1GB of limit https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_allowed_packet
I think this will not be a problema.

Reply:
This was my question, so I think there is no real limit to the number of elements in the in-clause: https://stackoverflow.com/questions/4275640/mysql-in-condition-limit/4275704